### ISSUE TITLE
Update Examples to use `const` inst `let`

### ISSUE THREAD

**Issue by FloorD** [k6io/docs#337]
Request: "Is there a reason the documentation uses `let` instead of `const` in the JavaScript examples? All of the examples I've read so far never reassign to the variables." https://k6io.slack.com/archives/C3FH0694P/p1622622359009900

Examples page:  https://k6.io/docs/examples/

**Comment by legander** [comment]
@ppcano @FloorD What do you think about having an eslint+prettier config specifically for the script examples?

**Comment by ppcano** [comment]
@legander We should. 


Ideally, we provide a command to lint all the examples in one folder. There is also the need to lint examples added within the content guides - we have to figure out how to handle this case.

Perhaps, it could be better to handle the `prettier` as a different task?

**Comment by legander** [comment]
I don't think it shouldn't be too much of an addition to create a barebones eslint + prettier config to use when going through the examples. Feels like a good opportunity to make it consistent when having to go through all the snippets and examples to change from let/var to const anyway?
CLI command for linting folder and inline content snippets sounds a little much perhaps. But in the interim maybe we could push the config files straight to the repo and document in the readme that these files should be used when adding script examples?

**Comment by amaaniqbal** [comment]
@FloorD @ppcano @legander Should I try updating the examples quickly for now?

Regarding the command to lint, `eslint` has some configurations around this. You can have a look at https://eslint.org/docs/rules/prefer-const for the same.
At the same time, these transformations are beyond the scope for `prettier`. Supporting content can be found at https://github.com/prettier/prettier/issues/5916#issuecomment-467742106/

**Comment by EdvinasDaugirdas** [comment]
@amaaniqbal,

Thank you for reaching out. Yes, indeed you're right, `prefer-const` is the rule that would have solved this particular issue!
Unfortunately, currently we don't have a good way of linting the script examples, so the solution is a little bit more involved. 

We have discussed the issue internally and already started working on it. 🎉
